cpg-language-python/src/main/kotlin/de/fraunhofer/aisec/cpg/frontends/python/StatementHandler.kt

```diff
@@ -276,6 +278,7 @@ class StatementHandler(frontend: PythonLanguageFrontend) :
         s: Python.ASTFunctionDef,
         recordDeclaration: RecordDeclaration? = null
     ): DeclarationStatement {
+        val language = language
```

Comment:
Why do we need this? Deleting this local copy doesn't seem to change anything.

Reply:
We need it so that Kotlin can smarrtcast "language" into `HasOperatorOverloading` below. I will try to investigate how to get rid of it.